usabilla-react-native/src/index.js

```diff
@@ -40,6 +60,8 @@ module.exports = {
     initialize,
     loadFeedbackForm,
     showLoadedForm,
+    sendEvent,
```

Comment:
my OCD triggers with non alphabetical functions ordering :)

Reply:
There is no rule to list functions alphabetically! they are listed in this order by functionality:
- Passive Feedback functions 
- Campaign functions
- Callbackfunctions